portal-web/test/functional/com/liferay/portalweb/macros/Clustering.macro

```diff
@@ -309,6 +309,8 @@ definition {
 			locator1 = "build-test.xml",
 			value1 = "wait-for-server-shutdown -Dapp.server.port.number=${nodePort}");
 
+		// Pausing 30 seconds due to LRQA-66618
```

Comment:
@catarineaguiar Is this pause actually needed? The ticket does not explain why, and I expect the server shutdown is completed already since the ant script is checking for the shutdown.